content/en/blog/2023/announcing-notation-rc5.md

```diff
@@ -0,0 +1,64 @@
+
+## Major updates in notation and notation-go
+
+Notation is a CLI tool to sign and verify OCI artifacts. The v1.0.0-RC.5 changes include but are not limited to the following. See the [v1.0.0-RC.5 release notes](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.5) for details.
+
+* Improve the output message when there is no signature associated with the image being referred
+* Add compatibility  with the [OCI distribution-spec 1.1.0-RC.2](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc2) and [OCI image-spec v1.1.0-RC.3](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc.3)
+* Use SHA256 instead of insecure SHA1 for certificate thumbprint in `notation inspect`
+* Update [OCI image-spec](https://github.com/opencontainers/image-spec) from 1.1.0-RC.2 to 1.1.0-RC.3
+
+Notation-go is a collection of Golang libraries to support signing and verifying OCI artifacts. It is based on the Notary Project signature specification. Notation-go v1.0.0-RC.5 removes OCI artifact manifest type from [Sign](https://pkg.go.dev/github.com/notaryproject/notation-go@main#Sign) function.
+
+### Deprecate flag `--signature-manifest` in notation sign
+
+Since v1.0.0-RC.5, Notation no longer supports creating signatures with OCI artifact manifest type in the signing process and deprecated the flag `--signature-manifest` from `notation sign`. Please note this is a breaking change in Notation v1.0.0-RC.5. We chose to make this deprecation before the stable release due to the upstream changes in the [OCI image-spec v1.1.0-rc.3](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc.3) that removed the artifact manifest.
+
+To mitigate the impact to Notation users, Notation v1.0.0-RC.5 is still able to read and verify the existing signatures stored with the OCI artifact manifest type in OCI v1.1.0-RC.1 compliant registries. OCI artifacts signed by earlier versions of Notation are still verifiable with Notation v1.0.0-RC.5.
+
+### Change default behavior and introduce a new flag `--allow-referrers-api`
+
+As Notation is close to the first stable release, we are trying to make Notation as stable as possible and avoid breaking changes going forward. Starting with Notation v1.0.0-RC.5 release, Notation will use the [Referrers Tag Schema](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#referrers-tag-schema) as the default behavior since most of the OCI registries are compatible with storing signatures using the Referrers Tag Schema.  The [Referrers API](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#enabling-the-referrers-api) and its fallback strategy are still under development according to recent OCI Distribution Spec changes, so it is marked as experimental in Notation v1.0.0-RC.5. 
```

Comment:
Again, don't use the words "stable" here. People can misconstrue it.  Just remove the this first sentence ~~"As Notation is close to the first stable release, we are trying to make Notation as stable as possible and avoid breaking changes going forward."~~  and just leave the remaining part as is
Starting with Notation v1.0.0-RC.5 release, Notation will use the [Referrers Tag Schema](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#referrers-tag-schema) as the default behavior since most of the OCI registries are compatible with storing signatures using the Referrers Tag Schema.  The [Referrers API](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#enabling-the-referrers-api) and its fallback strategy are still under development according to recent OCI Distribution Spec changes, so it is marked as experimental in Notation v1.0.0-RC.5.

Reply:
@iamsamirzon Please note that [Stable](https://en.wikipedia.org/wiki/Software_release_life_cycle#Stable_release) is an official term in Software release life cycle, which means the last release candidate (RC) that has passed all stages of verification and tests. The remaining bugs are considered acceptable. This release goes to [production](https://en.wikipedia.org/wiki/Deployment_environment#Production).